library/src/window_impl.cxx

```diff
@@ -50,10 +50,11 @@ class window_impl::internals
 
   std::unique_ptr<camera_impl> Camera;
   vtkSmartPointer<vtkRenderWindow> RenWin;
-  vtkSmartPointer<vtkF3DRenderer> Renderer;
+  vtkNew<vtkF3DRendererWithColoring> Renderer;
```

Comment:
So `vtkF3DRenderer` is never instantiated now? Then maybe we should merge it with `vtkF3DRendererWithColoring`

Reply:
> So vtkF3DRenderer is never instantiated now?

Correct

> Then maybe we should merge it with vtkF3DRendererWithColoring

I don't think that is worth it. I like the conceptual separation.

Reply:
fair enough